### ISSUE TITLE
Unidentified RemoteFortressReader segfault on 64-bit Linux

### ISSUE THREAD

**Issue by lethosor** [DFHack/dfhack#1071]
Originally reported at JapaMala/armok-vision#42, but Japa said it looked like a RemoteFortressReader issue on IRC.

**Comment by se5a** [comment]
I also got a segfault just recently not sure if it's related. 
iirc I was designating a large multi z level area to gather plants. 

here's the terminal output:
```$ sh dfhack
Loading bindings from data/init/interface.txt
Resetting textures
Warning: Plugin automaterial compiled for DFHack 0.43.05-beta1-11-g67f39316, running DFHack 0.43.05-beta1-0-g33060c7d
Warning: Plugin resume compiled for DFHack 0.43.05-beta1-11-g67f39316, running DFHack 0.43.05-beta1-0-g33060c7d
Warning: Plugin mousequery compiled for DFHack 0.43.05-beta1-11-g67f39316, running DFHack 0.43.05-beta1-0-g33060c7d
Warning: Plugin twbt compiled for DFHack 0.43.05-beta1-11-g67f39316, running DFHack 0.43.05-beta1-0-g33060c7d
TWBT: version 5.77
TWBT: no display patch (not an error)
DFHack is ready. Have a nice day!
DFHack version 0.43.05-beta1 (release) on x86_64
Type in '?' or 'help' for general help, 'ls' to see all commands.
Clearing all animal hospitals
[DFHack]# Segmentation faut
```

not sure if the "clearing all animal hospitals" there is related at all or not.

**Comment by lethosor** [comment]
That's almost certainly unrelated, unless you were using something like Armok Vision at the time.

**Comment by se5a** [comment]
it sounded like you were not 100% that it was infact RemoteFortressReader

**Comment by JapaMala** [comment]
Fixed in e60d8de
